irg_planetary_ephemeris/src/ephemeris_publisher_node.cpp

```diff
@@ -447,8 +462,9 @@ main(int argc, char *argv[])
 
     broadcast_transforms(broadcaster, sun_visibility_pub, reference_body,
                          mission_lat, mission_lon, mission_elev,
-                         target_bodies, current_time, ephemeris);
+                         target_bodies, current_time, timeOffset, ephemeris);
     
+    ros::spinOnce();
```

Comment:
Why did you add spinOnce? It's possible this is something we should have here and never realized it.